### PR TITLE
Feature/export ts interfaces

### PR DESCRIPTION
**Pull request by kwhitaker**
This exports the TS interfaces so they can be referenced in code that implements `useAxios`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,5 +139,5 @@ MIT
  [npm-url]: https://npmjs.org/package/@use-hooks/axios
  [download-image]: https://img.shields.io/npm/dm/@use-hooks/axios.svg?style=flat-square
  [download-url]: https://npmjs.org/package/@use-hooks/axios
- [travis-url]: https://travis-ci.com/use-hooks/react-hooks-axios
- [travis-image]: https://img.shields.io/travis/com/use-hooks/react-hooks-axios.svg?style=flat-square
```
[the remark this answers is not in the document]
Comment (kwhitaker) [reply README.md]: sorry about that - I'm not sure what caused that. I'll clean it up - thanks!

[the remark this answers is not in the document]
Comment (kwhitaker) [reply README.md]: Should be fixed now!

**Comment by int64ago** [comment]
please also fix the conflicts, thanks

**Comment by kwhitaker** [comment]
Conflicts should be resolved now.

**Comment by int64ago** [comment]
thanks

**Comment by int64ago** [comment]
v1.3.0

**Pull request merged**